### PR TITLE
Add missing translation for meta_title in Spree::Product

### PR DESCRIPTION
**Pull request by bricesanchez**
Could you backport this feature on 3-1-stable ?

### CODE REVIEW

File: db/migrate/20171003153758_add_translation_meta_title_to_product.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class AddTranslationMetaTitleToProduct < ActiveRecord::Migration[5.1]
```
Comment (damianlegawiec) [inline db/migrate/20171003153758_add_translation_meta_title_to_product.rb]: @bricesanchez please change the `ActiveRecord::Migration[5.1]` to `ActiveRecord::Migration[5.0]` or `ActiveRecord::Migration[4.2]` so tests will pass :)

**Comment by bricesanchez** [comment]
@damianlegawiec done :)

**Comment by damianlegawiec** [comment]
Thank you @bricesanchez 👍

**Comment by bricesanchez** [comment]
@damianlegawiec Is there a way to backport this feature on a Spree 3.1 with Rails 4?

**Pull request merged**